time crop_start_time 
time crop_end_time
int32 num_samples
string[] message_names
bool stop_streaming
---
task_recorder2_msgs/DataSample[] filtered_and_cropped_messages
task_recorder2_msgs/Description description
string info
int32 return_code
int32 SERVICE_CALL_FAILED = 0
int32 SERVICE_CALL_SUCCESSFUL = 1